components/member-card/card.module.scss

```diff
@@ -13,7 +13,8 @@
   
   box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
   border-radius: 5%;
-  transition: transform 0.2s ease-in-out;
+  // transition: transform 0.2s ease-in-out;
```

Comment:
Please remove this code. We do not need this.

Reply:
the commented one?

Reply:
yes